content/pages/index.rst

```diff
@@ -10,79 +10,57 @@ Home
 :save_as: index.html
 :sortorder: 1
 
+.. class:: hide-heading row align-items-center
 
-.. class:: container hide-heading
-
-#######
 Summary
-#######
-
-.. class:: hide-heading row align-items-center my-5 py-5
-
-Row
-===
-
-.. class:: col-5 mx-auto col-md-5 order-md-2
+=======
 
+.. class:: col-5 mx-auto col-md-5 order-md-2 container-fluid
 .. figure:: /images/cyclonedds.svg
 
-.. class:: col-md-7 order-md-1 text-center text-md-start pr-md-5
-
-Cyclone DDS
------------
-
-High performant and robust open source OMG DDS implementation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Brief summary of what makes Eclipse Cyclone DDS unique.
+.. class:: col-10  mx-auto col-md-5 order-md-2 text-center  pr-md-5
 
-.. class:: ato-button
+Building Portable & Interoperable Distributed Datacentric Systems
+-----------------------------------------------------------------
+Cyclone DDS is a high performing, OMG-DDS standard based data sharing technology which allows system designers to create digital twines of their systems’ entities to share their states, events, data-streams and messages on the network in real-time and fault-tolerant way
 
-`Get Started <https://github.com/eclipse-cyclonedds/cyclonedds>`_
-
-
-.. class:: container-fluid hide-heading
-
-##########
-Highlights
-##########
 
 .. class:: hide-heading highlights masthead-followup row m-0 border border-white
 
-Row
-===
+Highlights
+==========
 
 .. class:: col-12 col-md-4 p-3 p-md-5 bg-light border border-white
 
-Column 1
---------
-
-**feature 1**
-Description for feature 1
+Fast & Dependable
+-----------------
 
-**feature 2**
-Description for feature 2
+Cyclone DDS has an extremely low latency and high throughput, especially in rush environments and noisy networks where error transmission rates are relatively high.​
+Performance variability and jitters are minimum which make it suitable for real-time and mission critical applications.​
+Cyclone also has a low footprint making it suitable for both enterprise and embedded systems.​
```

Comment:
Please use *Cyclone DDS* consistently. I know everybody knows we mean Cyclone DDS when we say Cyclone but still. The official name is Cyclone DDS. (Should of course have been *Splice*, but still).